CovertInfiltration/Src/CovertInfiltration/Classes/X2StrategyElement_InfiltrationUnlocks.uc

```diff
@@ -50,18 +48,11 @@ static function X2SoldierUnlockTemplate CreateInfilSize2Unlock()
 	// Requirements
 	Template.Requirements.RequiredHighestSoldierRank = default.INFILTRATION_2_RANK;
 	Template.Requirements.bVisibleIfSoldierRankGatesNotMet = true;
-	Template.Requirements.SpecialRequirementsFn = Infiltration2UnlockFn;
 
 	// Cost
 	Resources.ItemTemplateName = 'Supplies';
-	Resources.Quantity = default.INFILTRATION_2_COST;
+	Resources.Quantity = 75;
```

Comment:
You are stacking costs on top of ones defined in config - just get rid of `Template.Cost` and `Template.Requirements` parts in code completely